modules/theora/video_stream_theora.cpp

```diff
@@ -351,8 +347,7 @@ void VideoStreamPlaybackTheora::set_file(const String &p_file) {
 	if (vorbis_p) {
 		vorbis_synthesis_init(&vd, &vi);
 		vorbis_block_init(&vd, &vb);
-		fprintf(stderr, "Ogg logical stream %lx is Vorbis %d channel %ld Hz audio.\n",
-				vo.serialno, vi.channels, vi.rate);
+		
```

Comment:
please remove tabs here

Reply:
Hi, what tabs are you referring that needs to be removed?

Reply:
at 312, 323, 350 lines.

Reply:
Done

Reply:
Thank you for reviewing this.